packages/create-fuels/test/e2e.test.ts

```diff
@@ -0,0 +1,65 @@
+import { safeExec } from '@fuel-ts/errors/test-utils';
+import { execSync } from 'child_process';
+
+import type { ProjectPaths } from './utils/bootstrapProject';
+import { bootstrapProject, resetFilesystem } from './utils/bootstrapProject';
+import { generateArgs } from './utils/generateArgs';
+import {
+  filterForcBuildFiles,
+  filterOriginalTemplateFiles,
+  getAllFiles,
+} from './utils/templateFiles';
+
+const { log } = console;
+
+const PUBLISHED_NPM_VERSION = process.env.PUBLISHED_NPM_VERSION;
+const programsToInclude = { contract: true, predicate: true, script: true };
+const availablePackages = ['pnpm'];
+
+/**
+ * @group e2e
+ */
+describe('`create fuels` package integrity', () => {
+  let paths: ProjectPaths;
+  let shouldSkip = false;
+
+  beforeAll(() => {
+    if (!PUBLISHED_NPM_VERSION) {
+      log('Skipping live `create fuels` test');
```

Comment:
It seems like this test would be skipped if the publishing fails because `PUBLISHED_NPM_VERSION` wouldn't be set, which is something that we don't want. Adding a check like `process.env.CI` would rule out that scenario:
```ts
if(!PUBLISHED_NPM_VERSION && !process.env.CI) {...} 
```

I don't know if the GitHub `CI` environment variable is available to node's `process.env` or you'd have to pass it.

Reply:
You're right, I didn't see this through completely.